scripts/5_acbm_to_matsim_xml.py

```diff
@@ -0,0 +1,163 @@
+    #  but not in the other. We will remove these people from the datasets.
+
+    individuals, activities, legs, legs_geo, households = filter_by_pid(
+        individuals, activities, legs, legs_geo, households
+    )
+
+    log_row_count(individuals, "individuals", "1_filter_by_pid", row_counts)
+    log_row_count(households, "households", "1_filter_by_pid", row_counts)
+    log_row_count(activities, "activities", "1_filter_by_pid", row_counts)
+    log_row_count(legs, "legs", "1_filter_by_pid", row_counts)
+    log_row_count(legs_geo, "legs_geo", "1_filter_by_pid", row_counts)
+
+    logger.info("2.3 - Rename geometry columns (for PAM)")
+    # TODO: Rename columns upstream in 3.3_assign_facility_all script
+    legs_geo.rename(
+        columns={
+            "start_location_geometry_wkt": "start_loc",
+            "end_location_geometry_wkt": "end_loc",
+        },
+        inplace=True,
+    )
+
+    logger.info("2.4 - Remove people with missing location data ")
+
+    individuals, households, activities, legs, legs_geo = filter_no_location(
+        individuals, households, activities, legs, legs_geo
+    )
+
+    log_row_count(individuals, "individuals", "2_filter_no_location", row_counts)
+    log_row_count(households, "households", "2_filter_no_location", row_counts)
+    log_row_count(activities, "activities", "2_filter_no_location", row_counts)
+    log_row_count(legs, "legs", "2_filter_no_location", row_counts)
+    log_row_count(legs_geo, "legs_geo", "2_filter_no_location", row_counts)
+
+    logger.info("2.5 - Log number of rows in each df after cleaning")
+
+    percentages = calculate_percentage_remaining(row_counts=row_counts)
+
+    # Log the percentages
+    for stage, df_name, count, percentage in percentages:
+        logger.info(
+            f"{df_name} - {stage} - {count} rows: {percentage:.1f}% rows remaining"
+        )
+
+    logger.info("3a - Convert geometry columns to POINT geometry")
+
+    # Function to convert to Point if not already a Point
+    def convert_to_point(value):
+        if isinstance(value, Point):
+            return value
+        return wkt.loads(value)
+
+    # Convert start_loc and end_loc to shapely point objects
+    legs_geo["start_loc"] = legs_geo["start_loc"].apply(convert_to_point)
+    legs_geo["end_loc"] = legs_geo["end_loc"].apply(convert_to_point)
+
+    # Convert to GeoDataFrame with start_loc as the active geometry
+    legs_geo = gpd.GeoDataFrame(legs_geo, geometry="start_loc")
+
+    logger.info("3b - Add home location to individuals")
+
+    # Apply
+    individuals_geo = add_home_location_to_individuals(legs_geo, individuals)
+
+    logger.info("4 - Write to MATSim XML")
+
+    logger.info("4.1 - Load travel diary to PAM")
+
+    population = load_travel_diary(
+        trips=legs_geo,
+        persons_attributes=individuals_geo,
+        tour_based=False,
+        include_loc=True,
+        sort_by_seq=True,
+        # hhs_attributes = None,
```

Comment:
Perhaps this commented argument can be removed? Alternatively could be added to the config if something that would be helpful to be configurable
```suggestion
```